widgetssdk/src/main/java/com/glia/widgets/messagecenter/MessageCenterView.kt

```diff
@@ -79,6 +81,21 @@ class MessageCenterView(
         initCallbacks()
     }
 
+    private fun setupViewAppearance() {
+        val callback: RequestCallback<Boolean> = RequestCallback { isAvailable, exception ->
+            if (exception != null) {
+                showUnexpectedErrorDialog()
+                return@RequestCallback
+            }
```

Comment:
Should this 'if' block also have 'sendMessageGroup.visibility = INVISIBLE' as in the next one?

Reply:
Also maybe it should initially be hidden and only shown once the network request confirms that the message center is available. Otherwise, it probably can be strange in case of a slow internet connection

Reply:
> Should this 'if' block also have 'sendMessageGroup.visibility = INVISIBLE' as in the next one?

@gugalo Sorry, haven't got the point. Maybe you mean VISIBLE? Could you please explain?

Reply:
> Also maybe it should initially be hidden and only shown once the network request confirms that the message center is available. Otherwise, it probably can be strange in case of a slow internet connection

@gugalo Yes, I also thought about this. We discussed this idea in [this thread ](https://salemove.slack.com/archives/C047N9EAG8Y/p1672413659817519)and on UI sync and it was decided that we will leave the initial approach hoping that integrators would not add Message Center view item if it's not available.

Reply:
> Should this 'if' block also have 'sendMessageGroup.visibility = INVISIBLE' as in the next one?

Ah, now I got your point. Thanks for pointing that out. Fixed 80f5688f298965532430aa2c36363cab8f6ac460